### ISSUE TITLE
"ng_file_name": add `ignorePrefix` option

### ISSUE THREAD

**Issue by alexlrobertson** [Gillespie59/eslint-plugin-angular#179]
As per "ng_service_name," services should be prefixed with a namespace. I think it makes sense to ignore the prefix from filenames.

Before:

```js
// asset.service.js
angular.module('xpAssetService', xpAssetService);
// Filename must be "xp-asset.service.js"  angular/ng_file_name
```

After:

```
// asset.service.js
angular.module('xpAssetService', xpAssetService);
```

**Comment by tilmanpotthof** [comment]
@alexlrobertson Do you think one prefix is enough, or might be a list of prefixes better?

**Comment by tilmanpotthof** [comment]
@alexlrobertson To prevent false positives I check that the next char is uppercase. Otherwise a prefix 'st' would be ignored for 'stAssetService' but also for 'stagingService' and report an expected filenane 'aging.service.js'.
